src/Nethermind/Nethermind.Serialization.Rlp/Nethermind.Serialization.Rlp.csproj

```diff
@@ -10,4 +10,8 @@
       <ProjectReference Include="..\Nethermind.Core\Nethermind.Core.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Nethermind.DotNetty.Buffers" Version="1.0.0" />
```

Comment:
Is it needed? Was it implicit before?

Reply:
Previously it was only in network package.

Reply:
Because now its in `Nethermind.Serialization.Rlp`.

Reply:
I mean `NettyRlpStream` is in it.